services/geocoding.py

```diff
@@ -45,7 +47,7 @@ def enhance(self, entries: List[PointOfInterest]) -> List[PointOfInterest]:
         coordinates: Mapping[str, Point] = {}
 
         for address in addresses_to_geocode:
-            geoData = make_geocode_request(address, gmaps)[0]["geometry"]["location"]  
+            geoData = self.make_geocode_request(address, gmaps)[0]["geometry"]["location"]  
             coordinates.update({address: Point(float(geoData["lat"]), float(geoData["lng"]))})
```

Comment:
What happens if geocoding fails? Does it raise an exception or just returns an empty geodata?
We should handle this, if we don't get actual coordinates we should not update coordinates dictionary

Reply:
add None check

Reply:
You're tested that is returns None if no coordinates found for an address?